src/core/thread/child_mask.hpp

```diff
@@ -58,9 +60,20 @@ namespace ot {
  * This class represents a bit-vector of child mask.
  *
  */
-class ChildMask
+class ChildMask : public Equatable<ChildMask>, public Clearable<ChildMask>
```

Comment:
Note that ChildMask has been changed to be an alias of `BitVector` in another PR #5267